Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx

```diff
@@ -253,16 +257,33 @@ void ITSMFTDeadMapBuilder::PrepareOutputCcdb(DataAllocator& output)
 
   info.setAdjustableEOV();
 
-  LOG(info) << "Sending object " << info.getPath() << "/" << info.getFileName()
-            << " of size " << image->size() << "bytes, valid for "
-            << info.getStartValidityTimestamp() << " : " << info.getEndValidityTimestamp();
+  if (ccdburl.empty()) {
 
-  if (mRunMFT) {
-    output.snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "TimeDeadMap", 1}, *image.get());
-    output.snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 1}, info);
-  } else {
-    output.snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "TimeDeadMap", 0}, *image.get());
-    output.snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 0}, info);
+    LOG(info) << "Sending object " << info.getPath() << "/" << info.getFileName()
+              << "to ccdb-populator, of size " << image->size() << "bytes, valid for "
+              << info.getStartValidityTimestamp() << " : " << info.getEndValidityTimestamp();
+
+    if (mRunMFT) {
+      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "TimeDeadMap", 1}, *image.get());
+      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 1}, info);
+    } else {
+      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "TimeDeadMap", 0}, *image.get());
+      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 0}, info);
+    }
+  }
+
+  else { // send from this workflow
+
+    LOG(info) << mSelfName << "sending object " << ccdburl << "/browse/" << info.getFileName()
+              << " of size " << image->size() << "bytes, valid for "
+              << info.getStartValidityTimestamp() << " : " << info.getEndValidityTimestamp();
+
+    o2::ccdb::CcdbApi mApi;
+    mApi.init(ccdburl);
+    mApi.storeAsBinaryFile(
+      &image->at(0), image->size(), info.getFileName(), info.getObjectType(),
+      info.getPath(), info.getMetaData(),
+      info.getStartValidityTimestamp(), info.getEndValidityTimestamp());
```

Comment:
Note that such an upload will neglect the AdjustableEOV flag. You may call 
```
o2::ccdb::adjustOverriddenEOV(mApi, info);
```
to enforce it.